gpii/node_modules/gpii-iniFile/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "gpii-iniFile",
```

Comment:
This won't be valid if this ever gets published to npm. Please rename here to gpii-ini-file and also in the filesystem

Reply:
done